### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- Include other technologies
- Modify css to include spacing, sizes, margins and paddings.

## Jira Story

- https://trello.com/c/N4pkrGe5

## Screenshot

image

### CODE REVIEW

File: mock-up/index.html [hunk 1]
```diff
@@ -23,12 +23,70 @@ <h1 class="title-words">Front-End Tools</h1>
       <p class="tech-release"><strong>Release: 2013</strong></p>
       <p class="tech-description">React is declarative, component based Javascript library for building user interfaces.</p>
     </div>
+
```
Comment (FrankIglesias) [inline mock-up/index.html]: remove extra lines

File: mock-up/index.html [hunk 2]
```diff
@@ -23,12 +23,70 @@ <h1 class="title-words">Front-End Tools</h1>
...
+    <div class="tech">
+      <img src="assets/angular_icon.svg" alt="angular" class="tech-img">
+      <p class="tech-name">AngularJS</p>
+      <p class="tech-release"><strong>Release: 2010</strong></p>
+      <p class="tech-description">AngularJS (commonly referred 1.X) is a Javascript framework for building SPAs with an MVC architechture.</p>
+    </div>
+
```
Comment (FrankIglesias) [inline mock-up/index.html]: same

File: mock-up/index.html [hunk 3]
```diff
@@ -23,12 +23,70 @@ <h1 class="title-words">Front-End Tools</h1>
...
+      <p class="tech-description">VueJS (pronunced /vju:/, like view) is an open-source progressive Javascript framework for building user interfaces.</p>
+    </div>
+
+    <div class="tech">
+      <img src="assets/elm_icon.svg" alt="elm" class="tech-img">
+      <p class="tech-name">Elm</p>
+      <p class="tech-release"><strong>Release: 2016</strong></p>
+      <p class="tech-description">Elm is a domain-specific functional programming language declaratively creating web browser-based graphical user interfaces.</p>
+    </div>
+
```
Comment (FrankIglesias) [inline mock-up/index.html]: same

File: mock-up/index.html [hunk 4]
```diff
@@ -23,12 +23,70 @@ <h1 class="title-words">Front-End Tools</h1>
...
   <div class="title">
     <img src="assets/triangle.svg" alt="triangle" class="title-img">
     <h1 class="title-words">Back-End Tools</h1>
   </div>
+
+  <div class="techs">
+    <div class="tech">
+      <img src="assets/rails_icon.png" alt="rails" class="tech-img">
+      <p class="tech-name">Ruby on Rails</p>
+      <p class="tech-release"><strong>Release: 2005</strong></p>
```
Comment (FrankIglesias) [inline mock-up/index.html]: use `font-weight: bold;` instead to remove the strong

File: mock-up/scss/tech.scss [hunk 5]
```diff
@@ -1,31 +1,33 @@
...
   border-radius: 5px;
   display: flex;
   flex-direction: column;
-  height: 325px;
+  height: 350px;
+  margin: 30px;
   padding: 10px;
   position: relative;
-  width: 200px;
+  width: 225px;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]:
```suggestion
  max-width: 225px;
  width: 100%;
```

File: mock-up/scss/tech.scss [hunk 6]
```diff
@@ -1,31 +1,33 @@
...
+  flex-wrap: wrap;
   margin: 0 150px;
-  padding: 30px 80px;
+  padding: 30px 80px
 }
 
 .tech {
   align-items: center;
-  background-color: #BED23C;
+  background-color: $tech-earls-green;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: use http://chir.ag/projects/name-that-color/

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: mock-up/index.html [hunk 7]
```diff
@@ -23,12 +23,70 @@ <h1 class="title-words">Front-End Tools</h1>
       <p class="tech-release"><strong>Release: 2013</strong></p>
```
Comment (betoobernal) [inline mock-up/index.html]: use css for strong style

File: mock-up/scss/colors.scss [hunk 8]
```diff
@@ -0,0 +1 @@
+$tech-earls-green: #BED23C
```
Comment (betoobernal) [inline mock-up/scss/colors.scss]: add an end line.

File: mock-up/scss/tech.scss [hunk 9]
```diff
@@ -11,29 +12,32 @@
...
+    width: 100%;
     z-index: -1;
   }
 }
 
 .tech-img {
   height: 100px;
-  object-fit: cover;
-  width: 120px;
+  object-fit: contain;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss:39]: cover

File: mock-up/scss/tech.scss [hunk 10]
```diff
@@ -1,8 +1,9 @@
 .techs-container {
   align-items: flex-start;
   display: flex;
+  flex-wrap: wrap;
   margin: 0 150px;
-  padding: 30px 80px;
+  padding: 30px 80px
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss:6]: missing semi colon

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by betoobernal — APPROVED** [review]

File: mock-up/scss/tech.scss (hunk 9 above)
Comment (joredudiaz) [reply mock-up/scss/tech.scss:39]: The resizing of the image with cover causes that the image won't fit in the width and height and the result is cut it

**Pull request merged**